### PR TITLE
Adds female patriotism (july in january edition)

### PR DESCRIPTION
**Pull request by MMMiracles**
I made the mistake of not fully supporting the fe(male) demographic of this game when I originally added patriotic apparel, so I've corrected my mistake.

:us: :us: :us: :us: :us: :us: :us: :us: 
patriotism
patriotismlong
:us: :us: :us: :us: :us: :us: :us: :us: 

:uk: :uk: :uk: :uk: :uk: :uk: :uk: :uk: 
britbong
britbonglong
:uk: :uk: :uk: :uk: :uk: :uk: :uk: :uk: 

:ru: :ru: :ru: :ru: :ru: :ru: :ru: :ru: 
communist
communistlong
:ru: :ru: :ru: :ru: :ru: :ru: :ru: :ru: 

:cl: MMMiracles
add: Three new bra sets have been added for the ladies out there who want to show their patriotism.
/:cl:

### CODE REVIEW

**Comment by francinum** [comment]
WILL conflict with #14987

**Comment by PKPenguin321** [comment]
:+1: :us:

**Comment by optimumtact** [comment]
:ru: :+1:

File: code/modules/mob/new_player/sprite_accessories.dm [hunk 1]
```diff
@@ -711,6 +711,21 @@
 	icon_state = "female_blackalt"
 	gender = FEMALE
 
+/datum/sprite_accessory/underwear/female_usastripe
+	name = "Ladies Freedom"
+	icon_state = "female_assblastusa"
```
Comment (neersighted) [inline code/modules/mob/new_player/sprite_accessories.dm]: kek

File: code/modules/mob/new_player/sprite_accessories.dm [hunk 2]
```diff
@@ -711,6 +711,21 @@
 	icon_state = "female_blackalt"
 	gender = FEMALE
 
+/datum/sprite_accessory/underwear/female_usastripe
+	name = "Ladies Freedom"
+	icon_state = "female_assblastusa"
+	gender = FEMALE
```
Comment (newfren) [inline code/modules/mob/new_player/sprite_accessories.dm]: Please remove this restriction.

File: code/modules/mob/new_player/sprite_accessories.dm [hunk 3]
```diff
@@ -711,6 +711,21 @@
...
+
+/datum/sprite_accessory/underwear/female_uk
+	name = "Ladies UK"
+	icon_state = "female_uk"
+	gender = FEMALE
+
+/datum/sprite_accessory/underwear/female_commie
+	name = "Ladies Commie"
+	icon_state = "female_commie"
+	gender = FEMALE
```
Comment (duncathan) [inline code/modules/mob/new_player/sprite_accessories.dm]: Make them neutral imo

Comment (MMMiracles) [inline code/modules/mob/new_player/sprite_accessories.dm]: guys already got their boxers, besides, it'd be a little inconsistent if only these were neutral and the rest of the sets weren't.

**Comment by Incoming5643** [comment]
A reminder that anyone can wear any underpants by visiting the dresser in the round.

The gender setting on these outfits are more to keep people from randomly generating men in babydolls.

**Comment by francinum** [comment]
@Incoming5643  False.

**Comment by Incoming5643** [comment]
oh was that just for socks?

**Comment by francinum** [comment]
All socks have no gender restrictions.

**Comment by Incoming5643** [comment]
yes but they used to, and during that time you could still put on the other gender's socks by using the dresser, I assumed underpants worked the same way

**Comment by francinum** [comment]
@Incoming5643 Only like, two socks ever had gender restrictions. and they were rather arbitrary at that.

**Comment by tkdrg** [comment]
conflicts

**Comment by MMMiracles** [comment]
i got this

**Comment by PKPenguin321** [comment]
usa
usa
usa
:us:

**Comment by Chocobro** [comment]
is that fucking russia underwear

**Comment by MMMiracles** [comment]
@Chocobro yes

**Pull request merged**